src/components/pages/Landing/RenderLandingPage.js

```diff
@@ -43,16 +44,16 @@ function RenderLandingPage(props) {
 
           {/* Button container */}
           <div className="landing-button-container">
-            <Button
-              classType="browse-btn btn"
-              buttonText="BROWSE PROGRAMS"
-              onClick={''}
-            ></Button>
-            <Button
-              classType="support-us-btn btn"
-              buttonText="SUPPORT US"
-              onClick={''}
-            ></Button>
+            <Link to="/browse-programs">
```

Comment:
I think a link is for navigation or a button is for actions. So since this is navigation, we could just add the styles to the link itself. That way when you use the tab key to move focus it will just be a single tab stop instead of two and it won't confuse the screen reader.